src/constants/ErrorType/UserError.ts

```diff
@@ -6,5 +6,13 @@ export default {
   "USER_NOT_FOUND": {
     code: 401000,
     msg: "text" // TODO: to be changed soon
+  },
+  "INVALID_TOKEN": {
```

Comment:
`TOKEN_INVALID`, I'd suggest that the subject (noun) comes first followed by verb (or adjective)